packages/front-end/components/Metrics/MetricForm/index.tsx

```diff
@@ -310,6 +311,10 @@ const MetricForm: FC<MetricFormProps> = ({
       minPercentChange: minPercentChange / 100,
     };
 
+    if (!edit && project) {
+      sendValue["projects"] = [project];
+    }
```

Comment:
We have a "duplicate metric" functionality now that uses this modal and populates `current`.  I don't think we want to set this to `[project]` if we're duplicating an existing metric.  We probably just want to copy over whatever is currently set (even if it's undefined).

Reply:
Sure, I added another variable `duplicate?: boolean = false` to track this, and have excluded sendValue when duplicating.